gflownet.py

```diff
@@ -462,16 +554,21 @@ def __init__(
                     self.model_path = Path(args.workdir) / args.gflownet.model_ckpt
             else:
                 self.model_path = args.gflownet.model_ckpt
-            if self.model_path.exists() and self.reload_ckpt:
-                self.model.load_state_dict(torch.load(self.model_path))
+            if (
+                self.model_path.exists()
+                and self.reload_ckpt
+                and self.forward_policy_type is not "uniform"
+            ):
+                self.forward_policy.model.load_state_dict(torch.load(self.model_path))
                 print("Reloaded GFN Model Checkpoint")
         else:
             self.model_path = None
         self.ckpt_period = args.gflownet.ckpt_period
         if self.ckpt_period in [None, -1]:
             self.ckpt_period = np.inf
-        self.model.to(self.device_torch)
-        self.target = copy.deepcopy(self.model)
```

Comment:
I did not understand what target was used for. It was in the previous code, and had been used when the condition `self.tau>0` was `True`. Since I wasn't sure what `tau` represents, I kept `self.target`.